GitHubService/GitHubService.API/Services/OrganizationService/OrganizationService.cs

```diff
@@ -0,0 +1,37 @@
+﻿using GitHubService.Domain.AggregatesModel.OrganizationAggregate;
+using Microsoft.Extensions.Logging;
+using RestSharp;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+
+namespace GitHubService.API.Services.OrganizationService
+{
+    public class OrganizationService : IOrganizationService
+    {
+        private readonly ILogger<OrganizationService> _logger;
+
+        public OrganizationService(ILogger<OrganizationService> logger)
+        {
+            _logger = logger;
+        }
+
+        /// <summary>
+        /// Call organization infroamtion from github api
+        /// </summary>
+        /// <returns>Organization Inforamtion</returns>
```

Comment:
I said earlier this comment is used as metadata for swagger.

Reply:
```suggestion
        /// <summary>
        /// Call organization information from GitHub API
        /// </summary>
        /// <returns>Organization Information</returns>
```

Reply:
1. 오타 수정, 
2. 메타 데이터를 Swagger 이 Controller 단에서만 사용 하는게 아니라면 설명이 가능할까요?
3. 이 메타 데이터는 어떻게 표시 되나요?

Reply:
https://docs.microsoft.com/ko-kr/aspnet/core/tutorials/getting-started-with-swashbuckle?view=aspnetcore-5.0&tabs=visual-studio

Reply:
wrong spell was changed.

Reply:
Good, 

![image](https://user-images.githubusercontent.com/41972657/132085217-9cd920f2-e005-4171-94af-131969c2b86c.png)

아마 아래 형태로 메타 데이터를 추가 해서 참고 하는 것 같은데,  정말 필요한 경우 Controller 아래의 Method 에 붙이면 될 것 같고. 

최대한 Project + Class + Method 의 이름으로 유추가 가능한 네이밍을 사용 하는 것이 좋다고 생각 됩니다. 
Organization Service (조직 서비스)

일단 조직 서비스라는 것도 어떤 도메인 영역을 가지고 있는지 현제 리뷰어가 잘 모르고 있으니, Description 부분에 추가 해 주면 감사하겠습니다!

Reply:
An 'Organization' is itself a domain. If you refer to the `Organization` item of github api, it will be easy to acquire domain knowledge.

https://docs.github.com/en/rest/reference/orgs#get-an-organization